pkg/api/login.go

```diff
@@ -258,10 +259,12 @@ func (hs *HTTPServer) loginUserWithUser(user *models.User, c *models.ReqContext)
 	}
 
 	hs.log.Debug("Got IP address from client address", "addr", addr, "ip", ip)
-	userToken, err := hs.AuthTokenService.CreateToken(c.Req.Context(), user.Id, ip, c.Req.UserAgent())
+	ctx := context.WithValue(c.Req.Context(), models.RequestURIKey{}, c.Req.RequestURI)
```

Comment:
Where is the `RequestURIKey` key/value pair on the context used?

Reply:
It's used in the associated Enterprise PR, maybe I should add it in the `RequestURIKey` comment?

Reply:
Aha, yeah it might be a good idea to add a comment so it doesn't look unused.

Reply:
I added the comment!

Reply:
Yeah I saw, thanks :)